src/test/resources/fixtures/hmh-ri/caliperEventGradeGraded.json

```diff
@@ -23,17 +23,17 @@
   "eventTime": "2016-11-15T10:57:06.000Z",
   "edApp": "https://www.hmhco.com/ReadingInventory",
   "generated": {
-    "id": "urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f", // ri-activity-id
+    "id": "urn:uuid:2050e852-5edb-4743-92d1-b53466de3a5f", // random UUID
     "type": "Score",
-    "attempt": "urn:uuid:c35603f7a3434e2cb74ef14d60e70f42", // ri-quiz-id
+    "attempt": "urn:uuid:635603f7a3434e2cb74ef14d60e70f42", // last attempt id
```

Comment:
points back to the same id used here https://github.com/hmhco/caliper-common-fixtures/pull/5/files#diff-cda1d6654d25effaed17317c5a5e8f44R11